docs/data/data-grid/filtering/filtering.md

```diff
@@ -203,6 +203,10 @@ The [`valueFormatter`](/x/react-data-grid/column-definition/#value-formatter) is
 If the column has a [`valueGetter`](/x/react-data-grid/column-definition/#value-getter), then `params.value` will be the resolved value.
 :::
 
+:::info
+The `getValueAsString` function allows you to convert the value of a filter item to a human-readable form before displayed in the UI of filter button tooltip.
```

Comment:
```suggestion
The filter button displays a tooltip on hover if there are active filters. Pass [`getValueAsString`](/x/api/data-grid/grid-filter-operator/) in the filter operator to customize or convert the value to a more human-readable form.
```

Reply:
Thanks for your edits and remarks. Text is fixed.